### PR TITLE
f-added quiz and quiz js

### PR DESCRIPTION
**Pull request by surya-nemiwal**
added a gk quiz page with javascript for that quiz

### CODE REVIEW

**Review by vivek-guda — COMMENTED** [review]
Looks good to go.

File: quizzes/gkquiz.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (bharati-21) [inline quizzes/gkquiz.html:6]: A suggestion - you can add meta author and description tag.

File: quizzes/quiz.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+let score = 0;
+
+loadQuiz();
+
+function loadQuiz() {
+  deselectAnswers();
+
+  const currentQuizData = quizData[currentQuiz];
+
+  questionEl.innerText = currentQuizData.question;
```
Comment (bharati-21) [inline quizzes/quiz.js:63]: Variable naming could be done better here. I'm not ab;e to understand their purpose. 👍🏽

**Review by bharati-21 — APPROVED** [review]
Dropped a few suggestions. You can look into that.

**Pull request merged**